src/main/java/org/jmhsrobotics/frc2024/RobotContainer.java

```diff
@@ -77,7 +77,7 @@ private void configureBindings() {
 	public Command getAutonomousCommand() {
 		Command picked = autoChooser.getSelected();
 		if (picked == null) {
-			return Commands.print("No autonomous command configured");
+			return new DriveTimeCommand(1.535, 0.3, driveSubsystem);
 		} else {
 			return picked;
 		}
```

Comment:
Should we out the command into the autoChooser as a default option, that was the default is visible via the driver dashboard?